.travis.yml

```diff
@@ -107,6 +107,38 @@ matrix:
       name: "Test blinky example - debug (NRF52840_DK)"
       env: EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=NRF52840_DK PROFILE=debug CACHE_NAME=debug-NRF52840_DK
 
+    - <<: *mbed-tools-test
```

Comment:
Was it intentional to not test `develop` profile with K64F, but only with the other boards?

Reply:
Added test for `develop` profile with `K64F`.